src/index.d.ts

```diff
@@ -98,19 +104,29 @@ type ConnectData = {
 }
 
 type TransactionsOptions = {
-  institutionId: string;
+  institutions?: Array<string>;
   from?: string;
   to?: string;
   investments?: string;
   last?: string;
   limit?: number;
+  institutionId?: string;
 }
 
 type TransactionOptions = {
-  institutionId: string;
   txId: string;
 }
 
+type PositionsOptions = {
+  institutions?: Array<string>;
+  groupIds?: Array<string>;
+  institutionIds?: Array<string>;
+  investmentIds?: Array<string>;
```

Comment:
Should it be `investments` to be consistent with the API, and with the TransactionsOptions above?